Automattic/wp-calypso
# Help center: entry point on mobile

#### Changes proposed in this Pull Request

Added the help center icon to the masterbar, behind a flag:
- In calypso, with the config `edit/help-center`
- Otherwise with check for `?enable-help-center` in URL
We register a plugin ('etk-help-center') in ETK and if we are on desktop, the help icon is added using `PinnedItems`, together with the HelpCenter component.
- If on mobile, the help icon is added to the masterbar. 

The icon on the masterbar communicates with the ETK plugin (where the HelpCenter component to show lives) through the sending of messages (`calipsoify` and `iframe-bridge-server`). 

#### Testing instructions



* Checkout branch
* Be sure to have your site, `public-api.wordpress.com` and `widgets.wp.com` sandboxed
* `yarn start` from calypso folder
* `yarn dev --sync` from `apps/editing-toolkit`
* `yarn dev --sync` from `apps/wpcom-block-editor`
* visit `http://calypso.localhost:3000/post/your-site.wordpress.com`
* Because the help center needs `enable-help-center` in ETK to work, it may be necessary for now to add it as a query parameter to the iframe url. I'll check if there is a easier way
<img width="536" alt="image" src="https://user-images.githubusercontent.com/52076348/163681187-a54bad04-10b2-495d-83ac-bfccfc6a99fe.png">




Related to #62702
Fixes #62702
